### PR TITLE
Updated Contributor names to sample json file

### PR DESCRIPTION
**Pull request by fabianwilliams**
Added Gladys and Fabian to the samples json file to allow for the adoption site to be populated

### CODE REVIEW

File: samples/nodejs-typescript-policies/env/.env.local [hunk 1]
```diff
@@ -4,9 +4,15 @@
 TEAMSFX_ENV=local
 APP_NAME=Policy Management
 DC_NAME=Policy copilot
-CONNECTOR_ID=policies
+CONNECTOR_ID=policies2
```
Comment (waldekmastykarz) [inline samples/nodejs-typescript-policies/env/.env.local:7]: Let's revert this change

File: samples/nodejs-typescript-policies/env/.env.local [hunk 2]
```diff
@@ -4,9 +4,15 @@
...
 CONNECTOR_NAME=Policy Management
 CONNECTOR_DESCRIPTION=Contains information about your policies.
 CONNECTOR_BASE_URL=https://localhost:3000/
 
-# Generated during provision, you can also add your own variables.
\ No newline at end of file
+# Generated during provision, you can also add your own variables.
```
Comment (waldekmastykarz) [inline samples/nodejs-typescript-policies/env/.env.local:12]: Let's revert these changes

**Review by waldekmastykarz — CHANGES_REQUESTED** [review]

**Comment by fabianwilliams** [comment]
resolved to reverted version
